cpp/bench.cxx

```diff
@@ -223,6 +221,182 @@ void vdot_f64c_blas(simsimd_f64_t const* a, simsimd_f64_t const* b, simsimd_size
 
 #endif
 
+namespace AVX512_harley_seal {
+
+uint8_t lookup8bit[256] = {
```

Comment:
Does it help to add const/constexpr? I wonder if it would encourage the table to be cached. It might also help to run a loop over it to pre-load it into cache too (although I figure prefetching would most likely get the whole table).

In my own experiments in the past, I did find the built in instructions to be faster, however.

Reply:
Assuming the size of the inputs - the tail will never be evaluated separately. I've just copied that part of the code for completeness.